### ISSUE TITLE
Make IsToggled change animated

### ISSUE THREAD

**Issue by zleao** [AndreiMisiukevich/TouchEffect#71]
Hi,
Would it be possible to have the ForceUpdateState in the IsToggledProperty with animate = true?
It would look something like this:

```
public static readonly BindableProperty IsToggledProperty = BindableProperty.CreateAttached(
            nameof(IsToggled),
            typeof(bool?),
            typeof(TouchEff),
            default(bool?),
            BindingMode.TwoWay,
            propertyChanged: (bindable, oldValue, newValue) =>
            {
                bindable.GetTouchEff()?.ForceUpdateState();
                TryGenerateEffect(bindable, oldValue, newValue);
            });
```

The reason is that this way I'm able to trigger an animation just by changing the `IsToggled` value...
Well, to be honest I need a bit of a 'trick' to get the animation to work... I change the state before changing the IsToggled, but the final result is exactly what I need.

So, bottom line, would it have impact if the `ForceUpdateState` in the `IsToggled ` was changed to `animate = true`?

Thanks for this awesome library!

**Comment by AndreiMisiukevich** [comment]
No, we cannot change it cuz it will lead us to wrong behavior for IsToggled + Ripple

But I added a backdoor for you

Write it somewhere in code and you will achieve what you need.
```csharp
TouchEff.IsForceUpdateStateAnimatedForIsToggledProperty = true;
```

**Comment by AndreiMisiukevich** [comment]
A new version will be available in 20-30 mins

**Comment by zleao** [comment]
Ah cool!
Thank you very much!
